docs/changelogs/CHANGELOG-2.0.md

```diff
@@ -2,6 +2,8 @@
 
 ## [2.0.1] YYYY-MM-DD
 
+- [#2870](https://github.com/epiphany-platform/epiphany/issues/2870) - Migration of OpenDistro for ElasticSearch
```

Comment:
I think that should go under Added or Updated tab

Reply:
That's true. My bad.